proto/engine/v1/fulu.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/pkg/errors"
 )
 
-func (ebe *ExecutionBundleFulu) GetDecodedExecutionRequests() (*ExecutionRequests, error) {
+func (ebe *ExecutionBundleFulu) GetDecodedExecutionRequests(maxDeposits, maxWithdrawals, maxConsolidations uint64) (*ExecutionRequests, error) {
```

Comment:
minor concern since all three parameters are of the same type, easy for a caller to accidentally pass them in the wrong order, go doesnt support named parameter. I think the only way is to use a struct. I dont have a strong feeling about this

Reply:
I don't mind adding a struct :+1: